pages/your_domain/plant_sciences.md

```diff
@@ -108,7 +108,7 @@ It is recommended that metadata collection is contemplated from the start of the
   * [e!DAL](https://edal.ipk-gatersleben.de/) is a free data management platform for which MIAPPE templates are in development.
   * The [ISA-Tools](https://isa-tools.org/) also include a configuration for MIAPPE and can be used both for filling-in metadata and for validating.
   * [Collaborative Open Plant Omics (COPO)](https://copo-project.org/) is a data management platform specific for the plant sciences.
-  * [FAIRsharing](https://fairsharing.org) is a manually curated registry of reporting guidelines, vocabularies, identifier scheme, models, formats, repositories, knowledge bases, and data policies that includes many resources relevant for managing plant phenotyping data.
+  * [FAIRsharing](https://fairsharing.org) is a manually curated registry of reporting guidelines, vocabularies, identifier schema, models, formats, repositories, knowledge bases, and data policies that includes many resources relevant for managing plant phenotyping data.
```

Comment:
"scheme" might be the correct words here. Apparently, "identification scheme" seems to be a thing.